cartridges/bm_adyen/site-template/meta/system-objecttype-extensions.xml

```diff
@@ -241,6 +241,14 @@
                 <externally-managed-flag>false</externally-managed-flag>
                 <min-length>0</min-length>
             </attribute-definition>
+            <attribute-definition attribute-id="AdyenShipBeforeDate">
```

Comment:
could you change this to AdyenShipBeforeDays rather then AdyenShipBeforeDate because it does not contain a data